src/routes/manage-selections/components/ProfessionItem.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactElement } from 'react'
```

Comment:
I created this ProfessionItem, which does only load a profession and then displays the Selection item. Same for the CustomDisciplineItem. 

We need this, because of non-conditional calling of hooks, but I am not sure, if this is the best solution. Other solution would be to create a hook, which load either a discipline or customDiscipline by evaluating if passed parameter is a string(=apikey) or number(=discipline).

Any arguments pro/con the solutions or even other solutions?

Reply:
@klinzo

Reply:
I did not like the idea of the hook, which checks for types, but you are right, just having a good Params-type and it is the better solution. 

I will not do this in this PR, as it is quite a lot of refactoring and has nothing to do with the real task of LUN-301. Created a new ticket for it (LUN-339) and will start working on it soon.